libs/orchestrator-ui-components/src/lib/components/SettingsPage/Status.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiSpacer,
+    EuiPanel,
+    EuiText,
+} from '@elastic/eui';
+import React, { FunctionComponent } from 'react';
+import { StatusDotIcon } from '../../icons';
+import { useEngineStatusQuery, useOrchestratorTheme } from '../../hooks';
+
+export const Status: FunctionComponent = () => {
+    const { theme } = useOrchestratorTheme();
+    const { data: engineStatus } = useEngineStatusQuery();
+    const engineStatusText: string = engineStatus?.global_status
+        ? `${engineStatus.global_status}`
+        : 'unavailable';
+    const isRunning = engineStatus?.global_status === 'RUNNING';
+
+    return (
+        <EuiPanel
+            hasShadow={false}
+            color="subdued"
+            paddingSize="l"
+            style={{ width: '50%' }}
+        >
+            <EuiFlexGroup>
+                <EuiFlexItem grow={false}>
+                    <EuiText size="s" style={{ fontWeight: 600 }}>
```

Comment:
Fontweights are part of the theme. You can use the value from the theme here instead of hardcoding.
`libs/orchestrator-ui-components/src/lib/theme/defaultOrchestratorTheme.ts`